applications/OptimizationApplication/tests/algorithm_tests/nlopt_tests/mma_shell_thickness_opt/optimization_parameters.json

```diff
@@ -93,7 +87,7 @@
                 "controls_lower_bound" : "0",
                 "controls_upper_bound" : "1",
                 "stopping_criteria"   : {
-                    "maximum_function_evalualtion": 2
+                    "maximum_function_evalualtion": 5
```

Comment:
I am not sure about this change, but is this intentional?

Reply:
Yes, to have more practical result